front/tests/utils/dataNormalizer.ts

```diff
@@ -23,7 +23,7 @@ export function cleanWhitespace(text: string = ''): string {
 }
 
 // Clean whitespace for an array of strings
-export function cleanWhitespaces(headers: string[]): string[] {
+export function cleanWhitespaceInArray(headers: string[]): string[] {
   return headers.map(cleanWhitespace);
```

Comment:
the `trim()` used on `text` in `cleanWhitespace()` seems useless, the regex in `replace()` is enough

Reply:
The `trim()` ensure that leading and trailing whitespace is always removed

Reply:
yes of course, but '\s+' ensure that every whitespaces are triggered no?

Reply:
Yes, but they will be replaced with a single whitespace, so adding `trim()` will ensure there is no whitespace at the beginning or the end

Reply:
aaaah yes ok